stablehlo/reference/InterpreterOps.cpp

```diff
@@ -135,6 +165,20 @@ SmallVector<InterpreterValue> evalRunParallelOp(
   return results;
 }
 
+llvm::Error evalProbeOp(InterpreterValue input, StringRef probeId,
+                        StringRef serializationDir) {
+  auto tensor = input.getTensor();
+  llvm::SmallString<128> filepath(serializationDir);
+  llvm::sys::path::append(filepath, probeId + ".npy");
+  numpy::serializeTensor(filepath, tensor.getType(), tensor.getData());
+
+  // After the tensor has been serialized to disk, append it to a metadata file
+  // to associate the serialized probe_id with the filepath. By default, this
+  // will live in an `index.csv` file generated in specified
+  // `probe_readout_dir`.
```

Comment:
Don't refer to the command line option name here, refer to input argument. Maybe rename input argument 

`serializationDir -> probeOutputDir`